mlir/lib/Dialect/Arith/Transforms/ExpandOps.cpp

```diff
@@ -228,9 +243,8 @@ struct BFloat16ExtFOpConverter : public OpRewritePattern<arith::ExtFOp> {
     Type operandETy = getElementTypeOrSelf(operandTy);
     Type resultETy = getElementTypeOrSelf(resultTy);
 
-    if (!operandETy.isBF16() || !resultETy.isF32()) {
+    if (!operandETy.isBF16() || !resultETy.isF32())
```

Comment:
Let's drop unrelated formatting/coding style changes from this PR and land this separately. This should take no time to review and make this PR shorter.